aws/table_aws_ssoadmin_permission_set.go

```diff
@@ -0,0 +1,241 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssoadmin"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsSsoAdminPermissionSet(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssoadmin_permission_set",
+		Description: "AWS SSO Permission Set",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"instance_arn", "arn"}),
```

Comment:
Question - why are both the `instance_arn` and the `arn` required as key columns? As best I could tell, the list hydrate seems to only use `instance_arn`?

Reply:
I thought the Get config was meant to allow Steampipe to retrieve a single row given the appropriate composite key - it makes a call to DescribePermissionSet instead of ListPermissionSets

e.g. I'd expect:

```sql
SELECT * FROM aws_sso_permission_set WHERE instance_arn = "..." AND arn = "..."
```

to sidestep a call to ListPermissionSets

Reply:
My mistake, you are absolutely right ... I'd confused the Get vs List key col sections ... thanks.